### PR TITLE
LIVE-558 Integrate new graphical assets for Nano S Plus

### PR DESCRIPTION
**Pull request by tlaforge-ledger**
## 🦒 Context (issues, jira)

https://ledgerhq.atlassian.net/browse/LIVE-558

## 💻  Description / Demo (image or video)

This pr adds all the assets and animations for the incoming product Ledger Nano S Plus. In order to remain compatible with the current design, this device will only be selectable in the onboarding if we start the application with the `SHOW_NANOSP=1` flag, if not present we should see a two option selector since we dropped the Blue.

If we start the application with the flag `DEBUG_LOTTIE=1` we will see an option to trigger a debug modal on the experimental settings screen. This modal should offer all combinations of animations for the three devices in both dark and light themes for easier testing.

There's another flag `OVERRIDE_MODEL_ID="nanoSP"` that allows us to replace the illustration (not the functionality) of the plugged in device in order to facilitate testing due to the shortage of prototypes for the upcoming device model. I expect there will be a need to regenerate screenshots after CI completes.

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

File: src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelector.js [hunk 1]
```diff
@@ -54,16 +55,16 @@ const devices = [
     label: "Nano S",
     Illu: NanoS,
   },
+  {
+    id: "nanoSP",
+    label: "nano S Plus",
```
Comment (juan-cortes) [inline src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelector.js]: **N**ano

File: src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ import Text from "~/renderer/components/Text";
 
 const DeviceIllustrationContainer = styled.div`
   position: absolute;
-  bottom: 0;
-  right: 0;
+  top: 93px;
```
Comment (juan-cortes) [inline src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js]: Why?

File: src/renderer/screens/manager/DeviceStorage/index.js [hunk 3]
```diff
@@ -24,17 +24,34 @@ import IconTriangleWarning from "~/renderer/icons/TriangleWarning";
 import IconCheckFull from "~/renderer/icons/CheckFull";
 
 import nanoS from "./images/nanoS.png";
+import nanoSDark from "./images/nanoS_dark.png";
```
Comment (juan-cortes) [inline src/renderer/screens/manager/DeviceStorage/index.js]: The images seem to be of different sizes, did you export x3 from lottie in all cases?
We don't want shitty rendering on retina

**Review by juan-cortes — CHANGES_REQUESTED** [review]
Few things to note, please add screenshots, a longer description (come on!, it's free) and a test plan for QA. Also, if you've done the debug screen explain how to access it (if accessible)

File: src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js (hunk 2 above)
Comment (tlaforge-ledger) [reply src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js]: Cause export are only the images and not the one with padding as before.
@juan-cortes Do you want I ask another export of images or is it ok like this ?

File: src/renderer/screens/manager/DeviceStorage/index.js (hunk 3 above)
Comment (tlaforge-ledger) [reply src/renderer/screens/manager/DeviceStorage/index.js]: I didn't export from lottie but from Figma.
I don't have the 4x retina rendering... So if it's shitty like this I can try another export too

File: src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js (hunk 2 above)
Comment (juan-cortes) [reply src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js:10]: As long as there's a reason and it doesn't break other things, I dont mind.

File: src/renderer/screens/manager/DeviceStorage/index.js (hunk 3 above)
Comment (juan-cortes) [reply src/renderer/screens/manager/DeviceStorage/index.js:27]: Yeah I meant figma sorry, but export in x3 and make sure all images are large. I'm sure the smalls ones in your commit are not going to show good on mac.

**Review by juan-cortes — DISMISSED** [review]
CI is still failing, remove all changes to nano S and nano X, then regenerate screenshots too.

**Comment by juan-cortes** [comment]
CI is passing, let's try generating screenshots 🤞🏼

**Comment by juan-cortes** [comment]
There's something wrong with the generated screenshots being in dark mode @nabil-brn can you take a look please?

**Review by juan-cortes — APPROVED** [review]
LGTM

**Review by valpinkman — APPROVED** [review]
Perfect. 

I like that you added a way to debug Lottie animations. Thanks 🙏🏻 

LGTM

**Review by grsoares21 — APPROVED** [review]
LGTM

**Pull request merged**